apprating/src/test/java/com/mindera/skeletoid/apprating/controller/AppRatingControllerTest.kt

```diff
@@ -0,0 +1,156 @@
+package com.mindera.skeletoid.apprating.controller
+
+import android.content.Context
+import com.mindera.skeletoid.apprating.callbacks.AppRatingDialogResponse
+import com.mindera.skeletoid.apprating.store.AppRatingStore
+import com.mindera.skeletoid.generic.DateUtils
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito
+import org.powermock.api.mockito.PowerMockito
+import org.powermock.core.classloader.annotations.PrepareForTest
+import org.powermock.modules.junit4.PowerMockRunner
+import java.util.Date
+import kotlin.test.assertTrue
+
+@RunWith(PowerMockRunner::class)
+@PrepareForTest(AppRatingStore::class, AppRatingController::class)
```

Comment:
out of curiosity - does jacoco report show coverage for this class? I had problems that it wasn't showing coverage if the class under test was being mocked (@PrepareForTest) with PowerMockito (so in this case AppRatingController)

Reply:
Yap, i checked now and the coverage appears as 0%. I read [here](https://github.com/powermock/powermock/wiki/Code-coverage-with-JaCoCo) that there's no workaround for now. Were you able to find a solution?

Reply:
Nope. The only way around this for me is just find a different approach without having to actually mock the class under test. Maybe we could move those tests to a different class just to have at least some coverage for the parts that don't require powermocking?

Reply:
we use powermockito in all of them in order to create the store 😞